tests/general/ImageDimensionBearTest.py

```diff
@@ -0,0 +1,91 @@
+import unittest
+from unittest.case import skip, skipIf
+from queue import Queue
+import shutil
+
+from bears.general.ImageDimensionBear import *
+from coalib.output.printers.LOG_LEVEL import LOG_LEVEL
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+from coalib.testing.BearTestHelper import generate_skip_decorator
+
+
+@generate_skip_decorator(ImageDimensionBear)
+class ImageDimensionBearTest(unittest.TestCase):
+    """
+    Runs unittests for ImageDimensionBear
+    """
+
+    def setUp(self):
+        self.section = Section('')
+        self.queue = Queue()
+        self.file_dict = {}
+        self.idb = ImageDimensionBear(None, self.section, self.queue)
+
+    def test_check_width(self):
+        with self.assertRaisesRegexp(ValueError, 'Width'):
```

Comment:
Please use the full error message `Width has to be greater than` to actually check if it's the actual error.